src/components/StablecoinAccordion/index.tsx

```diff
@@ -0,0 +1,274 @@
+import React, { ReactNode } from "react"
+import {
+  Accordion,
+  Box,
+  Flex,
+  Heading,
+  Icon,
+  LinkBox,
+  LinkOverlay,
+  Text,
+} from "@chakra-ui/react"
+import { MdArrowForward } from "react-icons/md"
+import ButtonLink from "../ButtonLink"
+import CardList from "../CardList"
+import InfoBanner from "../InfoBanner"
+import Link from "../Link"
+import Translation from "../Translation"
+import { AccordionCustomItem } from "./AccordionCustomItem"
+import { useStablecoinAccordion } from "./use-stablecoin-accordion"
+import { TranslationKey } from "../../utils/translations"
+
+export type ChildOnlyType = {
+  children: ReactNode
+}
+
+const SectionTitle = (props: ChildOnlyType) => (
+  <Heading
+    as="h4"
+    fontSize="1.25rem"
+    fontWeight={700}
+    lineHeight="22px"
+    textAlign="left"
+    mt={0}
+    {...props}
+  />
+)
+
+const StepBoxContainer = (props: ChildOnlyType) => (
+  <Box mt={4} mb={8} {...props} />
+)
+
+const StepBox = (
+  props: { to: string } & Record<"titleId" | "descId", TranslationKey>
+) => (
+  <Flex
+    as={LinkBox}
+    alignItems={{ base: "flex-start", md: "normal" }}
+    background="background"
+    border="1px"
+    borderColor="border"
+    color="text"
+    flexDirection={{ base: "column", md: "row" }}
+    p={4}
+    sx={{
+      "&:not(:first-of-type)": {
+        mt: "-1px",
+      },
+    }}
+    _hover={{
+      background: "ednBackground",
+      transition: "transform 0.2s",
+      transform: "scale(1.05)",
+    }}
+  >
+    <Flex justifyContent="space-between" alignItems="center" width="100%">
+      <Box>
+        <LinkOverlay
+          as={Link}
+          color="inherit"
+          textDecoration="inherit"
+          to={props.to}
+          fontWeight={700}
+          textAlign="left"
+          _hover={{
+            textDecoration: "inherit",
+          }}
+        >
+          <Translation id={props.titleId} />
+        </LinkOverlay>
+        <Text mb={0}>
+          <Translation id={props.descId} />
+        </Text>
+      </Box>
+      <Icon as={MdArrowForward} ml={4} minW={6} />
+    </Flex>
+  </Flex>
+)
+
+const H4 = (props: ChildOnlyType) => (
+  <Heading fontSize="1.25rem" fontWeight={700} mt={0} mb={4} {...props} />
+)
+
+export interface IProps {}
+
+const StablecoinAccordion: React.FC<IProps> = () => {
+  const { cardListGroups } = useStablecoinAccordion()
+
+  return (
+    <Accordion
+      borderRadius="base"
+      mt={12}
+      width="full"
+      allowToggle
+      reduceMotion
+    >
+      <AccordionCustomItem category="dapps">
+        {({ LeftColumnPanel, RightColumnPanel }) => (
```

Comment:
Wondering why you have chosen this approach instead of just define those components in this file as you did with `StepBoxContainer` for example.

I mean, why we can't just define
```tsx
const LeftColumnPanel = (props: ChildOnlyType & Partial<BoxProps>) => ();
const RightColumnPanel = (props: ChildOnlyType) => ();
```
in this file?

I think that if there is no special reason, this would simplify the structure of the component.

Reply:
@pettinarip it was an approach to contain these to components within `AccordionCustomItem`, but I can understand the inconsistency you describe here.

No reason not to use the panel components directly instead of through a child function. I'll go ahead and make the change. 😄

Reply:
Cool. Yea, this is just a minor comment but I think simplicity will make it more readable for other devs. Thanks for doing that.